server/tailsql/tailsql_test.go

```diff
@@ -15,10 +15,13 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"net/url"
+	"path/filepath"
 	"regexp"
 	"strings"
 	"testing"
 
+	setecclient "github.com/tailscale/setec/client/setec"
```

Comment:
Why not just `setec`? The package path is structured so that setec.Blah reads nicely and makes it clear what the thing is.

Reply:
Leftover from a previous version that is no longer needed; fixed.